test/integration/src/test/java/io/pravega/test/integration/MetricsTest.java

```diff
@@ -187,10 +189,11 @@ public void metricsTimeBasedCacheEvictionTest() throws Exception {
             log.info("Creating Reader group : {}", readerGroupName1);
 
             readerGroupManager.createReaderGroup(readerGroupName1,
-                                                 ReaderGroupConfig.builder()
-                                                                  .stream(Stream.of(scope, STREAM_NAME))
-                                                                  .automaticCheckpointIntervalMillis(2000)
-                                                                  .build());
+                    ReaderGroupConfig
```

Comment:
This change seems not necessary, as the line is not exceeding the max length.

Reply:
The continuation indent should be 8 only.